client/my-sites/hosting/main.js

```diff
@@ -5,7 +5,87 @@
  */
 
 import React from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+import { localize } from 'i18n-calypso';
 
-const Hosting = () => <p>Placeholder</p>;
+/**
+ * Internal dependencies
+ */
+
+import Main from 'components/main';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import DocumentHead from 'components/data/document-head';
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import Button from 'components/button';
+import MaterialIcon from 'components/material-icon';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const DataSection = ( { title, data } ) => {
+	return (
+		<p>
+			<strong>{ title }: </strong>
+			{ data }
+		</p>
+	);
+};
+
+const Hosting = ( { translate } ) => {
+	const dummyInfo = {
+		[ translate( 'URL' ) ]: 'sftp1.wordpress.com',
+		[ translate( 'Port' ) ]: 22,
+		[ translate( 'Username' ) ]: 'test',
+		[ translate( 'Password' ) ]: 'test',
+	};
+
+	return (
+		<Main className="hosting is-wide-layout">
+			<PageViewTracker path="hosting/:site" title="Hosting" />
+			<DocumentHead title={ translate( 'Hosting' ) } />
+			<SidebarNavigation />
+			<div className="hosting__cards">
+				<Card>
+					<div className="hosting__card-icon-col">
+						<MaterialIcon className="hosting__card-icon" icon="cloud" size={ 48 } />
+					</div>
+					<div className="hosting__card-col">
+						<CardHeading>{ translate( 'SFTP Information' ) }</CardHeading>
+						<p>
+							{ translate( "Access and edit your website's files directly using an FTP client." ) }
+						</p>
+						{ map( dummyInfo, ( data, title ) => (
+							<DataSection { ...{ data, title } } />
```

Comment:
When we iterate over items, we need a unique key. https://reactjs.org/docs/lists-and-keys.html#keys

We have a static order, so index should be okay.

<img width="676" alt="Screen Shot 2019-10-02 at 12 39 15 PM" src="https://user-images.githubusercontent.com/1270189/66076149-02c2be80-e512-11e9-910e-32f2b20c74a2.png">